src/game/protos/object.h

```diff
@@ -239,20 +239,26 @@ void object_set_pal_limit(object *obj, int limit);
 int object_get_pal_limit(const object *obj);
 
 vec2i object_get_size(const object *obj);
+vec2i object_get_fpos(const object *obj);
 vec2i object_get_pos(const object *obj);
 vec2f object_get_vel(const object *obj);
 
+void object_set_fpos(object *obj, vec2i pos);
 void object_set_pos(object *obj, vec2i pos);
 void object_set_vel(object *obj, vec2f vel);
 
 int object_w(const object *obj);
 int object_h(const object *obj);
+int object_fpx(const object *obj);
 int object_px(const object *obj);
+int object_fpy(const object *obj);
 int object_py(const object *obj);
 float object_vx(const object *obj);
 float object_vy(const object *obj);
 
+void object_set_fpx(object *obj, int val);
 void object_set_px(object *obj, int val);
+void object_set_fpy(object *obj, int val);
 void object_set_py(object *obj, int val);
 void object_set_vx(object *obj, float val);
 void object_set_vy(object *obj, float val);
```

Comment:
please add comments to explain what distinguishes the many similarly named functions here.